probe_scraper/check_repositories.py

```diff
@@ -11,6 +11,7 @@
 REPOSITORIES = os.path.join(
     os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "repositories.yaml"
 )
+EXPECTED_MISSING_FILES = {("firefox-desktop", "gfx/metrics.yaml")}
```

Comment:
Should we document this somewhere?

How will this behave when the file will be added again? (it's missing because the related patch was backed out due to performance problems that are being worked on)

Reply:
it won't break anything, but this part should be removed when the file returns.